plugins/systemstat.koplugin/main.lua

```diff
@@ -56,60 +68,57 @@ end
 
 local function systemInfo()
     local result = {}
-    do
-        local stat = io.open("/proc/stat", "r")
-        if stat ~= nil then
```

Comment:
I guess this was to have a local scope so at the end of the do..end the `local stat` memory is dropped, so we don't use more memory when reading meminfo next.
But these are at most 2KB, so I guess it doesn't really matter as this is rarely called - so I'm fine with removing this strange constructs :)

Reply:
There might have been a shadow variable down the line at one point in time ;).